src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
+import { Post } from '../../types/post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title, body, user, comments,
+  } = post;
```

Comment:
```suggestion
  const {
    title,
    body, 
    user, 
    comments,
  } = post;
```
If there are more than 3 props, then it's better to provide a new line for each of them

Reply:
fixed